azurelinuxagent/pa/rdma/centos.py

```diff
@@ -155,7 +155,7 @@ def install_rdma_drivers(self, fw_version):
 
         # Install kernel mode driver (kmod-microsoft-hyper-v-rdma-*)
         kmod_pkg = self.get_file_by_pattern(
-            pkgs, "%s-(\d+\.){3,}(%s)-\d{8}\.x86_64.rpm" % (self.rdma_kernel_mode_package_name, fw_version))
+            pkgs, "%s-(\\d+\\.){3,}(%s)-\\d{8}\\.x86_64\\.rpm" % (self.rdma_kernel_mode_package_name, fw_version))
```

Comment:
why not a raw string?

Reply:
It could have been raw, but it's still valid as is.  This issue was identified with python 3.8.